src/lib/intercom/branding.js

```diff
@@ -14,19 +14,22 @@ const handle = async event => {
   if (!iframe) {
     return
   }
-  let branding = config.get('branding')
+  let branding = config.get(BRANDING_KEY)
   if (!branding) {
     branding = await fetchBranding()
+    config.put(BRANDING_KEY)
   }
 
-  iframe.contentWindow.postMessage(
-    {
-      namespace: 'etvas',
-      event: 'response.cssVars',
-      payload: branding
-    },
-    '*'
-  )
+  if (branding) {
```

Comment:
is this the else branch for the previous if statement ?

Reply:
No, the branding can still be null.